test/blob_test.js

```diff
@@ -0,0 +1,67 @@
+var azure   = require('../');
+var assert  = require('assert');
+var uuid    = require('uuid');
+
+// Create azure blob client
+var blob = new azure.Blob({
+  accountId: process.env.AZURE_STORAGE_ACCOUNT,
+  accessKey: process.env.AZURE_STORAGE_ACCESS_KEY
+});
+
+describe.only('Azure Blob Container', function() {
+
+  it('create container with metadata', function() {
+    var containerName = uuid.v4();
```

Comment:
I would recommend against random names... use the a fixed name, that way it's easier to cleanup... Or use a prefix and do a search at the end to delete everything with the given prefix...